README.md

```diff
@@ -319,4 +319,26 @@ In this section, we will guide you through the steps to add new icons. By follow
 
 8. **Updating Ion Icons**
    * Install the latest version of Ion Icons by running the following command:
-   `npm i ionicons@7`
\ No newline at end of file
+   `npm i ionicons@7`
+
+### How to create a FAQ?
+In this section, we will guide you through the steps for creating a new FAQ.
+
+1. **Navigate to the _faqs directory:**
+   * In the site's main directory, open the _faqs folder. This is where all your FAQ posts are stored.
+
+2. **Create a New Markdown File:**
+   * Create a new file in the _faqs folder.
```

Comment:
You should also indicate that you must append with `.md`.

As for prepending with Q1, Q2, Q3, I'm unsure if we really want to keep them numbered as they are now. We may need to shuffle the order later. We should think more about that.

Reply:
For example, we may want to organize by topic, if these files can include multiple questions.